lib/signaling/v2/peerconnection.js

```diff
@@ -370,13 +370,13 @@ class PeerConnectionV2 extends StateMachine {
 
     encodingParameters.on('changed', this._onEncodingParametersChanged);
 
-    peerConnection.onconnectionstatechange = this._handleConnectionStateChange.bind(this);
-    peerConnection.ondatachannel = this._handleDataChannelEvent.bind(this);
-    peerConnection.onicecandidate = this._handleIceCandidateEvent.bind(this);
-    peerConnection.oniceconnectionstatechange = this._handleIceConnectionStateChange.bind(this);
-    peerConnection.onicegatheringstatechange = this._handleIceGatheringStateChange.bind(this);
-    peerConnection.onsignalingstatechange = this._handleSignalingStateChange.bind(this);
-    peerConnection.ontrack = this._handleTrackEvent.bind(this);
+    peerConnection.addEventListener('connectionstatechange', this._handleConnectionStateChange.bind(this));
+    peerConnection.addEventListener('datachannel', this._handleDataChannelEvent.bind(this));
+    peerConnection.addEventListener('icecandidate', this._handleIceCandidateEvent.bind(this));
+    peerConnection.addEventListener('iceconnectionstatechange', this._handleIceConnectionStateChange.bind(this));
+    peerConnection.addEventListener('icegatheringstatechange', this._handleIceGatheringStateChange.bind(this));
+    peerConnection.addEventListener('signalingstatechange', this._handleSignalingStateChange.bind(this));
+    peerConnection.addEventListener('track', this._handleTrackEvent.bind(this));
```

Comment:
I made this change in the `feature/webrtc-redirection`, but now I think it is better to revert it.